### PR TITLE
Replace Speaker <Input/> with tailwind

### PR DESCRIPTION
**Pull request by evanugarte**

### CODE REVIEW

File: src/Pages/Speaker/Speaker.js [hunk 1]
```diff
@@ -60,12 +60,12 @@ function SpeakersPage(props) {
     <div>
       <div className='flex justify-center pt-10'>
         <div className="md:w-1/2">
-          <Input placeholder='Enter YouTube Link' onChange=
+          <input placeholder='Enter YouTube Link' onChange=
             {(e) => setUrl(e.target.value)}
-          className="sign-input"
+          className="sign-input indent-2"
```
Comment (joshua-demo) [inline src/Pages/Speaker/Speaker.js:65]: className="sign-input indent-2 w-full h-8"

File: src/Pages/Speaker/Speaker.js [hunk 2]
```diff
@@ -60,12 +60,12 @@ function SpeakersPage(props) {
...
           style={{ width: '100%', height: '2rem' }}
```
Comment (joshua-demo) [inline src/Pages/Speaker/Speaker.js:66]: remove this due to using tailwind

**Review by NicholasLe04 — APPROVED** [review]
looks good

File: src/Pages/Speaker/Speaker.js (hunk 1 above)
Comment (evanugarte) [reply src/Pages/Speaker/Speaker.js]: done

**Review by steeevin88 — APPROVED** [review]
looks good

File: src/Pages/Speaker/Speaker.js [hunk 3]
```diff
@@ -60,12 +60,12 @@ function SpeakersPage(props) {
     <div>
       <div className='flex justify-center pt-10'>
         <div className="md:w-1/2">
-          <Input placeholder='Enter YouTube Link' onChange=
+          <input placeholder='Enter YouTube Link' onChange=
             {(e) => setUrl(e.target.value)}
-          className="sign-input"
+          className="sign-input indent-2 w-full h-8"
           style={{ width: '100%', height: '2rem' }}
```
Comment (evanugarte) [inline src/Pages/Speaker/Speaker.js:66]:
```suggestion
```

File: src/Pages/Speaker/Speaker.js (hunk 2 above)
Comment (evanugarte) [reply src/Pages/Speaker/Speaker.js]: done

**Review by joshua-demo — APPROVED** [review]
lgtm

**Review by ahmadgaz — COMMENTED** [review]
looks good!

**Review by ahmadgaz — COMMENTED** [review]
looks good!

**Pull request merged**